label_sync/labels.md

```diff
@@ -0,0 +1,124 @@
+| <a id="close/needs-information" href="#close/needs-information">`close/needs-information`</a> | TODO, see [kubernetes/test-infra#7251](https://github.com/kubernetes/test-infra/issues/7251)| humans | |
+| <a id="close/not-reproducible" href="#close/not-reproducible">`close/not-reproducible`</a> | TODO, see [kubernetes/test-infra#7251](https://github.com/kubernetes/test-infra/issues/7251)| humans | |
+| <a id="close/unresolved" href="#close/unresolved">`close/unresolved`</a> | TODO, see [kubernetes/test-infra#7251](https://github.com/kubernetes/test-infra/issues/7251)| humans | |
+| <a id="committee/conduct" href="#committee/conduct">`committee/conduct`</a> | Denotes an issue or PR intended to be handled by the (as of yet non-existent) code of conduct committee.| anyone |  [label](https://git.k8s.io/test-infra/prow/plugins/label) |
+| <a id="committee/steering" href="#committee/steering">`committee/steering`</a> | Denotes an issue or PR intended to be handled by the kubernetes steering committee.| anyone |  [label](https://git.k8s.io/test-infra/prow/plugins/label) |
+| <a id="kind/bug" href="#kind/bug">`kind/bug`</a> | Categorizes issue or PR as related to a bug.| anyone |  [label](https://git.k8s.io/test-infra/prow/plugins/label) |
+| <a id="kind/documentation" href="#kind/documentation">`kind/documentation`</a> | Categorizes issue or PR as related to documentation.| anyone |  [label](https://git.k8s.io/test-infra/prow/plugins/label) |
+| <a id="kind/feature" href="#kind/feature">`kind/feature`</a> | Categorizes issue or PR as related to a new feature.| anyone |  [label](https://git.k8s.io/test-infra/prow/plugins/label) |
+| <a id="lifecycle/frozen" href="#lifecycle/frozen">`lifecycle/frozen`</a> | Indicates that an issue or PR should not be auto-closed due to staleness. <br><br> This was previously `keep-open`, | anyone |  [lifecycle](https://git.k8s.io/test-infra/prow/plugins/lifecycle) |
+| <a id="lifecycle/rotten" href="#lifecycle/rotten">`lifecycle/rotten`</a> | Denotes an issue or PR that has aged beyond lifecycle/stale and will be auto-closed.| fejta-bot or anyone |  [lifecycle](https://git.k8s.io/test-infra/prow/plugins/lifecycle) |
+| <a id="lifecycle/stale" href="#lifecycle/stale">`lifecycle/stale`</a> | Denotes an issue or PR has remained open with no activity for too long and has become stale. <br><br> This was previously `stale`, | anyone |  [lifecycle](https://git.k8s.io/test-infra/prow/plugins/lifecycle) |
+| <a id="priority/awaiting-more-evidence" href="#priority/awaiting-more-evidence">`priority/awaiting-more-evidence`</a> | Lowest priority. Possibly useful, but not yet enough support to actually get it done. These are mostly place-holders for potentially good ideas, so that they don't get completely forgotten, and can be referenced /deduped every time they come up.| anyone |  [label](https://git.k8s.io/test-infra/prow/plugins/label) |
```

Comment:
This really isn't a priority.. more a triaging label

Reply:
Perhaps, but I'm not interested in changing label content within this PR.  This is copy pasta from https://github.com/kubernetes/community/blob/c86f809e9fe7bcff4c1486f1573d636b63faf152/contributors/guide/issue-triage.md